docs/user-guide/content-video-toolbar.md

```diff
@@ -0,0 +1,23 @@
+# Video Content Toolbar
+**********************
+
+As soon as a video content is added, the Video Content Toolbar appears on top of the video: 
+
+<img src="../img/content-video-toolbar/c-video-toolbar.jpg" class="ms-docimage"/>
+
+Through this toolbar, the story editor is able to perform the following operation:
+
+* **Change media source** <img src="../img/button/change-media2.jpg" class="ms-docbutton"/> accessing the [Media Editor](media-editor-window.md)
+
+* **Mute video** <img src="../img/button/mute-video-button.jpg" class="ms-docbutton"/> to disable the video audio
+
+* **Enable Autoplay** <img src="../img/button/autoplay-button.jpg" class="ms-docbutton"/> to play the video automatically once the user is on it
+
+* **Enable Loop** <img src="../img/button/loop-botton.jpg" class="ms-docbutton"/> to continuously repeat the video
+
+* **Hide caption** button <img src="../img/button/hide-caption.jpg" class="ms-docbutton"/> to show/hide the description under the image. (It is available if the description is added to the image on the [Media Editor](media-editor-window.md))
```

Comment:
* **Hide caption** button <img src="../img/button/hide-caption.jpg" class="ms-docbutton"/> to show/hide the description under the **video**: this button is present only if a description has been provided for the video resource (see the [Media Editor](media-editor-window.md) tool for example).